tests/Records/NestedFlattenObject2.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Crell\Serde\Records;
+
+use Crell\Serde\Attributes\Field;
+
+class NestedFlattenObject2
```

Comment:
Better class name, please.

Reply:
I am a bit out of idea here, tried `ObjectWithFlattenedNestedFlattenObject`